src/content/docs/infrastructure/host-integrations/host-integrations-list/apache-druid-integration.mdx

```diff
@@ -0,0 +1,266 @@
+Filepath
+            </th>
+            <th>
+Code snippet
+            </th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>
+`PATH/TO/broker.runtime.properties`
+            </td>
+            <td>
+```yml
+# Monitoring
+druid.monitoring.monitors=["org.apache.druid.java.util.metrics.JvmMonitor"]
+druid.emitter=prometheus
+druid.emitter.logging.logLevel=info
+druid.emitter.prometheus.strategy=exporter
+druid.emitter.prometheus.port=19091
+```
+            </td>
+        </tr>
+        <tr>
+            <td>
+`PATH/TO/Coordinator-Overlord.runtime.properties`
+            </td>
+            <td>
+```yml
+# Monitoring
+druid.monitoring.monitors=["org.apache.druid.java.util.metrics.JvmMonitor"]
+druid.emitter=prometheus
+druid.emitter.logging.logLevel=info
+druid.emitter.prometheus.strategy=exporter
+druid.emitter.prometheus.port=19092
+```
+            </td>
+        </tr>
+        <tr>
+            <td>
+`PATH/TO/Historical.runtime.properties`
+            </td>
+            <td>
+```yml
+# Monitoring
+druid.monitoring.monitors=["org.apache.druid.java.util.metrics.JvmMonitor"]
+druid.emitter=prometheus
+druid.emitter.logging.logLevel=info
+druid.emitter.prometheus.strategy=exporter
+druid.emitter.prometheus.port=19093
+```
+            </td>
+        </tr>
+        <tr>
+            <td>
+`PATH/TO/MiddleManager.runtime.properties`
+            </td>
+            <td>
+```yml
+# Monitoring
+druid.monitoring.monitors=["org.apache.druid.java.util.metrics.JvmMonitor"]
+druid.emitter=prometheus
+druid.emitter.logging.logLevel=info
+druid.emitter.prometheus.strategy=exporter
+druid.emitter.prometheus.port=19094
+```
+            </td>
+        </tr>
+        <tr>
+            <td>
+`PATH/TO/Router.runtime.properties`
+            </td>
+            <td>
+```yml
+# Monitoring
+druid.monitoring.monitors=["org.apache.druid.java.util.metrics.JvmMonitor"]
+druid.emitter=prometheus
+druid.emitter.logging.logLevel=info
+druid.emitter.prometheus.strategy=exporter
+druid.emitter.prometheus.port=19095
+```
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+    </Step>
+    <Step>
+## Install the Prometheus Emitter Extension [#prometheus-emitter-extension]
+
+1. Run the following commands to create a folder named `prometheus-emitter` within the `extensions` folder directory of your Apache Druid setup:
+```shell
+cd apache-druid-29.0.0/extensions/
+```
+```shell
+sudo mkdir prometheus-emitter
+```
+2. Navigate to the druid download directory and run the following command to create jar files that will be called when the server starts up:
+```shell
+java \
+  -cp "lib/*" \
+  -Ddruid.extensions.directory="extensions" \
+  -Ddruid.extensions.hadoopDependenciesDir="hadoop-dependencies" \
+  org.apache.druid.cli.Main tools pull-deps \
+  --no-default-hadoop \
+  -c "org.apache.druid.extensions.contrib:prometheus-emitter:24.0.0"
+```   
+    </Step>
+    <Step>
+## Configure `nri-prometheus` for Apache Druid [#configure]
```

Comment:
for steps related to new relic specific files/configurations, i might add a brief, standard/repeatable explanation about how this yml file organizes the directory or something to that effect. it doesn't have to be verbose or anything, maybe something like 'Create a file named `nri-prometheus-config.yml`. This file directs X to collect Y metrics, so it becomes part of our ingest pipeline.' idk how it works i'm just riffing

Reply:
Great idea, but this will have to be a fast follow. I'll consult with SMEs!